examples/react-whiteboard/package.json

```diff
@@ -9,8 +9,8 @@
     "eject": "react-scripts eject"
   },
   "dependencies": {
-    "@liveblocks/client": "^0.17.11",
-    "@liveblocks/react": "^0.17.11",
+    "@liveblocks/client": "^0.18.0-beta3",
+    "@liveblocks/react": "^0.18.0-beta3",
```

Comment:
I just published 0.18.0-beta4, which hopefully is the final version that we can launch as 0.18.0 next week. Could you upgrade once more? Sorry 'bout the hassle!